cfn_sweeper/base/cfn_resources.py

```diff
@@ -77,3 +78,26 @@ def is_managed_by_cloudformation(physical_resource_id: str, resource_array: list
             return True
     else:
         return False
+
+
+ 
+def get_stacks(resource_array: dict) -> list:
```

Comment:
lets discuss this implementation - my main issue being that we are already collecting this information in https://github.com/rileydakota/cfn-sweeper/blob/e50ac6646030b10382023678a47f89f0860fb1c5/cfn_sweeper/base/cfn_resources.py#L26 - we just may need to "finesse" the existing implementation a bit to get what we want. It feels messy recalling the same APIs we have already called just to get a different piece of information when we can just persist it on the first call instead